test/integration/dockerregistryclient_test.go

```diff
@@ -224,14 +222,19 @@ func doTestRegistryClientImage(t *testing.T, registry, version string) {
 	}
 }
 
+func TestRegistryClientImageV2ManifestV2(t *testing.T) {
+	t.Skip("openshift/schema-v2-test-repo is being pushed by Docker 1.10, which causes this test to fail because digests don't match")
```

Comment:
Why would this fail? The importer should be able to import manifest V2 schema 2 just fine both by tag and digest.